projects/1.20/assets/crabbers-delight/crabbersdelight/lang/zh_cn.json

```
@@ -0,0 +1,121 @@
+{
+  "itemGroup.crabbersdelight": "蟹农乐事",
+
+  "item.crabbersdelight.crab": "螃蟹",
+  "item.crabbersdelight.cooked_crab": "熟螃蟹",
+
+  "item.crabbersdelight.clawster": "龙虾",
+  "item.crabbersdelight.cooked_clawster": "熟龙虾",
+
+  "item.crabbersdelight.shrimp": "虾",
+  "item.crabbersdelight.cooked_shrimp": "熟虾",
+
+  "item.crabbersdelight.clam": "蛤蜊",
+  "item.crabbersdelight.raw_clam_meat": "生蛤蜊肉",
+  "item.crabbersdelight.cooked_clam_meat": "熟蛤蜊肉",
+
+  "item.crabbersdelight.raw_squid_tentacles": "生鱿鱼须",
+  "item.crabbersdelight.cooked_squid_tentacles": "熟鱿鱼须",
+
+  "item.crabbersdelight.raw_glow_squid_tentacles": "生发光鱿鱼须",
+  "item.crabbersdelight.cooked_glow_squid_tentacles": "熟发光鱿鱼须",
+
+  "item.crabbersdelight.raw_frog_leg": "生青蛙腿",
+  "item.crabbersdelight.cooked_frog_leg": "熟青蛙腿",
+
+  "item.crabbersdelight.kelp_shake": "海带奶昔",
+  "item.crabbersdelight.crab_cakes": "蟹肉饼",
+  "item.crabbersdelight.crab_legs": "蟹腿",
+  "item.crabbersdelight.cooked_tropical_fish": "熟热带鱼",
+  "item.crabbersdelight.shrimp_skewer": "烤虾串",
+  "item.crabbersdelight.surf_and_turf": "海陆大餐",
+  "item.crabbersdelight.fish_stick": "烤鱼柳",
+  "item.crabbersdelight.clam_bake": "蛤蜊大杂烩",
+  "item.crabbersdelight.clam_chowder": "蛤蜊浓汤",
+  "item.crabbersdelight.tropical_fish_slice": "热带鱼片",
+  "item.crabbersdelight.cooked_tropical_fish_slice": "熟热带鱼片",
+  "item.crabbersdelight.pufferfish_slice": "河豚片",
+  "item.crabbersdelight.cooked_pufferfish_slice": "熟河豚片",
+  "item.crabbersdelight.bisque": "法式贝类浓汤",
+  "item.crabbersdelight.seafood_gumbo": "海鲜秋葵浓汤",
+  "item.crabbersdelight.stuffed_nautilus_shell": "填馅鹦鹉螺酿",
```

```suggestion
  "item.crabbersdelight.stuffed_nautilus_shell": "填馅鹦鹉螺",
```
“填陷xx”跟“xx酿”应该是一个意思？